exercises/js/blog.js

```diff
@@ -0,0 +1,25 @@
+class RevealText {
+  constructor(options) {
+    this.blogList = $(options.listSelector);
+  }
+
+  init() {
+    this.bindClickTolist();
+  }
+
+  bindClickTolist() {
+    this.blogList.delegate("h3", "click", function(event) {
+      event.preventDefault();
+      let h3Element = $(event.target.closest("h3")),
+          excerpt = h3Element.next(),
+          visibleExcerpt = h3Element.parents("ul").find("p:visible");
+      visibleExcerpt.slideUp();    
+      excerpt.slideDown();
+    });
+  }
+}
+$(document).ready(() => {
+  const options = { listSelector: "[data-id='blog'] ul" },
```

Comment:
This should be a variable, not const.
Also, pass the DOM element to the class instead of selector.

If we pass `$("#blog h3")` to the class, the code in the click event method can become much shorter and cleaner. Also, use `slideToggle` and method chaining in the click event method to make it look better.